src/XMakeCommandLine/UnitTests/XMake_Tests.cs

```diff
@@ -699,7 +698,19 @@ private void RobustDelete(string path)
             }
         }
 
-#if FEATURE_RUN_EXE_IN_TESTS
+        /// <summary>
+        /// Run msbuild.exe within corerun.exe
+        /// It concatenates the given msbuild path to the given parameters and sends it to corerun.
+        /// </summary>
+        private string runMSBuildWithinCoreRun(string pathToMsBuildExe, string msbuildParameters, bool expectSuccess = true)
```

Comment:
.NET coding style is for methods to start with a capital letter, so this should be RunMSBuildWithinCoreRun.